lib/index.js

```diff
@@ -96,14 +104,14 @@ exports.plugin = {
             method: req.method, // e.g GET/POST
             url: request.url.path, // the path the person requested
             headers: request.raw.req.headers, // all HTTP Headers
-            info: request.info, // all additional request info (useful for debug)
-            auth: request.auth, // any authentication details e.g. the decoded JWT
+            info: request.info, // all additional request info (useful to debug)
```

Comment:
```suggestion
            info: request.info, // all additional request info (useful for debugging)
```

Reply:
I know this goes onto the next line but does make more sense in English and doesn't skip to the next line in the rich diff.

Reply:
80 character limit ... requires scrolling horizontal in my code editor. 😮

Reply:
Hm, do you want to change it back before I merge?